codyze-console/src/main/webapp/src/app.css

```diff
@@ -1,3 +1,10 @@
 @import 'tailwindcss';
 @plugin '@tailwindcss/forms';
 @plugin '@tailwindcss/typography';
+
+@theme {
+  /*  --font-sans: InterVariable, sans-serif;*/
```

Comment:
```suggestion
```

Reply:
Ah sorry this needs to be re-enabled again